### PR TITLE
Release version 1.1.0

### PR DESCRIPTION
**Pull request by avani-bhojwani**
Main updates include:
* [PR #14](https://github.com/nf-core/denovotranscript/pull/14) - Template update for nf-core/tools v3.0.2
* [PR #15](https://github.com/nf-core/denovotranscript/pull/15) - Fix logic for skip_assembly parameter
* [PR #16](https://github.com/nf-core/denovotranscript/pull/16) - Increase resources for Trinity
* [PR #21](https://github.com/nf-core/denovotranscript/pull/21) - Fix salmon quantification in skip_assembly mode

## PR checklist

- [X] This comment contains a description of changes (with reason).
- [N/A] If you've fixed a bug or added code that should be tested, add tests!
- [N/A] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/denovotranscript/tree/master/.github/CONTRIBUTING.md)
- [N/A] If necessary, also make a PR on the nf-core/denovotranscript _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [X] Make sure your code lints (`nf-core lint`).
- [X] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [X] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [N/A] Usage Documentation in `docs/usage.md` is updated.
- [N/A] Output Documentation in `docs/output.md` is updated.
- [X] `CHANGELOG.md` is updated.
- [N/A] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,15 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.1.0 - [2024-10-25]
```
Comment (FriederikeHanssen) [inline CHANGELOG.md:6]:
```suggestion
## v1.1.0 - [2024-11-06]
```

File: nextflow.config [hunk 2]
```diff
@@ -280,43 +266,46 @@ manifest {
     homePage        = 'https://github.com/nf-core/denovotranscript'
     description     = """A pipeline for de novo transcriptome assembly of short reads from bulk RNA-seq"""
     mainScript      = 'main.nf'
-    nextflowVersion = '!>=23.04.0'
-    version         = '1.0.0'
+    nextflowVersion = '!>=24.04.2'
+    version         = '1.1.0'
     doi             = ''
```
Comment (FriederikeHanssen) [inline nextflow.config:271]: doi

Comment (FriederikeHanssen) [inline nextflow.config:271]:
```suggestion
    doi             = '10.5281/zenodo.13324371'
```

**Review by FriederikeHanssen — APPROVED** [review]
Looks great 🚀

**Review by FloWuenne — APPROVED** [review]
LGTM! 🚀

**Pull request merged**